content/guides/accessibility/color-considerations.mdx

```diff
@@ -0,0 +1,144 @@
+
+### The APCA
+
+The [Advanced Accessible Perceptual Contrast Algorithm](https://www.myndex.com/APCA/) (<abbr>APCA</abbr>) is a proposal and is **not normative**. In terms of WCAG, and therefore, MAS-C compliance, this means that color contrast is valid **only** if it meets WCAG 2.1 criteria. 
+
+A color value that is APCA-compliant **may not** be WCAG compliant until APCA is integrated into WCAG 3.0. WCAG 3.0 is still a work in progress and currently does not have an anticipated release date.
+
+## Compliance 
+
+The following guidance is based on one or more **normative** Web Content Accessibility Guidelines (<abbr>WCAG</abbr>) success criterion. This means that it is objective requirements with implications that affect MAS-C compliance efforts, as well as international laws regarding accessibility.
+
+Non-normative suggestions are also supplied, but marked as such.
+
+## Visual themes
+
+GitHub offers two parent themes: Day (light) and Night (dark). Each parent theme has a default appearance, and then a suite of sub-themes. Sub-themes include considerations for high and low contrast experiences, as well as certain vision conditions.
+
+It is possible to set GitHub to use a dark sub-theme when a parent Day theme has been set, and vice-versa. This allows a user to have a fine-grained control over their experience.
+
+### Default theme
+
+GitHub uses the default Day (light) theme when a user visits the site in a logged out state. Because the visitor is logged out, they are unable to adjust the theme in the [Appearance settings](https://github.com/settings/appearance).
+
+Because of this, [the default Day theme **must** to satisfy WCAG) criteria for color-related concerns](https://github.com/github/accessibility/blob/main/docs/coaching-recommendations/primer/color-modes.md#guidance). From a high-level, this means that **all** GitHub content displayed using the default Day theme must:
+
+- Pass color contrast requirements, and
+- Have the mechanism for accessing and changing the theme be fully WCAG compliant.
+
+This is the strongest guarantee we have that: 
+
+1. Content can be perceived—and therefore understood and used—across a wide range of vision conditions, and that
+2. The mechanism to adjust GitHub's theme can be operated by the widest range of input modes.
+
+This allows users to use GitHub in a logged-out state. If a user is logged in, it also enables them to address their own specific usability and access needs, as well as aesthetic desires.
+
+### Non-default themes
+
+[All sub-themes **do not** need to meet WCAG color contrast requirements](https://github.com/github/accessibility/blob/main/docs/coaching-recommendations/primer/color-modes.md#guidance), provided the default Day theme and its content is accessible. This guidance is derived from [WCAG 2.1 Technique G174](https://www.w3.org/WAI/WCAG21/Techniques/general/G174).
```

Comment:
I feel like we talk about everything but Dark mode, but it's not made clear if that's needs to be compliant? I get that the sub themes don't have to be, which is fine.

Reply:
Agree, we should add it here.

```suggestion
[All sub-themes **do not** need to meet WCAG color contrast requirements](https://github.com/github/accessibility/blob/main/docs/coaching-recommendations/primer/color-modes.md#guidance), provided the default Day and Night theme and its content is accessible. This guidance is derived from [WCAG 2.1 Technique G174](https://www.w3.org/WAI/WCAG21/Techniques/general/G174).
```

Reply:
Holding off on this one until we have consensus in https://github.com/primer/design/pull/560#discussion_r1301903858